Installation/cmake/modules/FindSuiteSparse.cmake

```diff
@@ -0,0 +1,437 @@
+## CMake file to locate SuiteSparse and its useful composite projects
```

Comment:
Yes, as soon as that file is copied somewhere (maybe with the name `FindSuiteSparse.cmake-license.md`?) the license terms are mostly fulfilled. In particular, the second clause
> Redistributions in binary form must reproduce the above copyright notice

 is not an issue, because a binary form of CGAL will never ship any part of CMake scripts.